examples/pony/celsius/Makefile

```diff
@@ -11,7 +11,7 @@ $(abspath $(lastword $(MAKEFILE_LIST)))_MK := 1
 TEST_TARGET := false
 
 # uncomment to disable generate pony related targets (build/test/clean) for pony sources in this directory
-#PONY_TARGET := false
+# PONY_TARGET := false
 
```

Comment:
It would be ideal to remove/undo any changes similar to this that aren't changing behavior in this and other `Makefile`s.